//
// Copyright 2019 Asylo authors
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//     http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.
//

syntax = "proto2";

package asylo;

import "asylo/crypto/certificate.proto";

// Configuration for an SGX AGE remote assertion authority.
message SgxAgeRemoteAssertionAuthorityConfig {
  // The root CAs' certificates in X.509 format. Must not be empty.
  repeated Certificate root_ca_certificates = 1;

  // The address and port of the remote assertion generator. Required.
  optional string server_address = 2;
}
